tests/rpm_tests.sh

```diff
@@ -1,8 +1,16 @@
 #!/bin/sh
 
+# If $top_srcdir has not been set by automake, import the test environment
+if [ -z "$top_srcdir" ]; then
+    top_srcdir="$(realpath $(dirname "$0")/..)"
+    . ${top_srcdir}/tests/testenv.sh
+fi
+
 # If no tests were selected by user or makefile, select all of them
-if [ $# -eq 0 ] && [ -z $RPM_TESTS_ARGS ]; then
+if [ $# -eq 0 ]; then
     set -- "${top_srcdir}"/tests/rpm_tests
 fi
 
-exec python3 -m nose -v --exclude=logpicker -a \!acceptance,\!slow $RPM_TESTS_ARGS "$@"
+# TODO: RPM_TESTS_ARGS are not usable in the container environment. Do we want to keep it?
```

Comment:
Have we used it for anything so far? I think no?

Reply:
Yeah, I'm convinced to remove this but I don't want to do it in this PR.